### ISSUE TITLE
Volume goes back to 200

### ISSUE THREAD

**Issue by yeongaori** [Zerthox/BetterDiscord-Plugins#137]
### Bug description

When I change the volume over 200, the volume goes back to 200 without clicking the slider a while after 

### Plugin

BetterVolume

### Plugin version

v2.3.0

### Discord client

Stable

**Comment by Zerthox** [comment]
So far I've not been successful with reproducing this. Resetting to 200% sounds a bit like it might be a cap introduced by Discord in some way. The plugin itself should only ever reset to 100%.

Any additional information on when this bug occurs?

**Comment by Jacajack** [comment]
As @KuchiSofts said, it just resets on its own. Interestingly, I got it to reset to 1% a couple of times, but I can't reproduce that behavior anymore.

![Peek 2023-01-19 19-39](https://user-images.githubusercontent.com/6173262/213531787-66a1c7e5-b517-4e91-a513-2ccecf8c855c.gif)

**Comment by Jacajack** [comment]
I am on Manjaro Linux KDE (kernel `5.15.85-1-MANJARO`) with `discord 0.0.24-1` and `betterdiscord-installer 1.2.0-1`. As far as I can tell, this issue wasn't present before Discord 0.0.24. If this could help, I can downgrade to `0.0.23`, `0.0.22` or `0.0.21` and verify that.

**Comment by Zerthox** [comment]
Looks like Discord's media engine on Linux saves the set volumes remotely, not locally like Windows. Volumes higher than 200% fail the validation and get reset. Nothing I can do about that for now, this is on Discord. Maybe a hack that only changes the volumes locally could bypass their validation.

**Comment by Zerthox** [comment]
Yes my assumption about it being platform specific is incorrect. I checked in the client and can confirm this is an experiment called "Remote Audio Settings". I will see whether it is feasible to force disable the experiment when the plugin runs. Only thing left is to hope they do not push this experiment through in its current state and limit volumes.

**Comment by Zerthox** [comment]
Here is an experimental version that attempts to disable the experiment: https://gist.github.com/Zerthox/d51daa11afb7682ba9ca7cdd190bf928. Let me know how it goes.

**Comment by Jacajack** [comment]
I did a quick test and it seems to be working well. I'll keep using it and let you know if I experience any issues

**Comment by Jacajack** [comment]
The volume indeed stays at higher values now, but despite setting it very loud (like 2000%), it appears to be internally capped at around 400%. At certain point it just doesn't get any louder. I don't know if this is related to this issue, though. Maybe it's always been this way and I just never noticed it - not sure.

**Comment by Jacajack** [comment]
Also, the volume goes back to 200% once Discord is restarted.

**Comment by yeongaori** [comment]
It doesn't go back to 200% for me on the experimental version. Even after restarting discord.

**Comment by Zerthox** [comment]
There's likely a race condition there. If Discord manages to load the remotely saved volumes before the plugin is started it will overwrite the locally saved volumes.

**Comment by iiQuiet** [comment]
> Here is an experimental version that attempts to disable the experiment: https://gist.github.com/Zerthox/d51daa11afb7682ba9ca7cdd190bf928. Let me know how it goes.

How do i use this?

**Comment by Nightfury58** [comment]
> Here is an experimental version that attempts to disable the experiment: https://gist.github.com/Zerthox/d51daa11afb7682ba9ca7cdd190bf928. Let me know how it goes.

hi, on windows 10 and can confirm that the plugin repo version resets to 200. this alternative experimental build does seem to work and not get reset, although its effects past 200% are... minimal

**Comment by firingthoughts** [comment]
im having the same problem, on windows 10, latest discord version, the experimental version here works so far. thank you

**Comment by demonkingzaine** [comment]
Experimental version stopped working today for me personally.

**Comment by yeongaori** [comment]
I'm having some crashes recently too
After recovering from the crash, the volume resets to 100 or 200, but it doesn't reset before it crashes.

**Comment by Reenuip** [comment]
I haven't noticed any crashing related to this, but I've also been having the volume reset back to 200% even with v2.3.2-rc.1
Strangely, if I toggle the plugin off and back on, my custom value sticks! Hopefully this workaround helps some of you having issues and shouldn't be needed anymore soon.

**Comment by JuniorGenius** [comment]
> I haven't noticed any crashing related to this, but I've also been having the volume reset back to 200% even with v2.3.2-rc.1 Strangely, if I toggle the plugin off and back on, my custom value sticks! Hopefully this workaround helps some of you having issues and shouldn't be needed anymore soon.

Same here

**Comment by Zerthox** [comment]
Need some adjustments for the BD site. Reopning until its approved.

**Comment by DatDuckSaysQuack** [comment]
Funny timing, was this resolved with the current version that is downloadable? Because I just grabbed this like 5 minutes ago and am encountering this bug.

**Comment by Zerthox** [comment]
> Funny timing, was this resolved with the current version that is downloadable? Because I just grabbed this like 5 minutes ago and am encountering this bug.

The version linked above in the conversion as well as the version in this repository force disable the experiment. The version downloadable from the BetterDiscord site does not (yet).

**Comment by JuniorGenius** [comment]
The issue is back

**Comment by nedet** [comment]
been getting this issue for a bit now
